src/tests/test_dal.py

```diff
@@ -0,0 +1,32 @@
+from polydatum import DataAccessLayer, DataManager as _BaseDataManager
```

Comment:
nitpick: Calling this `_BaseDataManager` is confusing in isolation. We do this in our own modules so we can re-use the name `DataManager`. The test does not do that.